dist/screeps.d.ts

```diff
@@ -1102,7 +1102,7 @@ interface FindPathOpts {
      */
     maxRooms?: number;
 }
-interface MoveToOpts {
+interface MoveToOpts extends FindPathOpts {
```

Comment:
This formulation matches the manual:
http://support.screeps.com/hc/en-us/articles/203013212-Creep#moveTo

Under the definition of of `opts`, it explicitly states:
> Any options supported by Room.findPath method.

So: any code which is relying on MoveToOpts not containing members from FindPathOpts is wrong.